### PR TITLE
Geolocate endpoint, tests and sample app

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: pangea-sdk/service/ip_intel/api.go [hunk 1]
```diff
@@ -40,6 +40,40 @@ func (e *IpIntel) Lookup(ctx context.Context, input *IpLookupRequest) (*pangea.P
 	return &panresp, err
 }
 
+// IP address geolocate
```
Comment (r3dcrosse) [inline pangea-sdk/service/ip_intel/api.go:43]:
```suggestion
// Geolocate an IP
```

File: pangea-sdk/service/ip_intel/api.go [hunk 2]
```diff
@@ -40,6 +40,40 @@ func (e *IpIntel) Lookup(ctx context.Context, input *IpLookupRequest) (*pangea.P
...
+//
+// Geolocate an IP address.
```
Comment (r3dcrosse) [inline pangea-sdk/service/ip_intel/api.go:45]:
```suggestion
// Retrieve information about the location of an IP address.
```

File: pangea-sdk/service/ip_intel/api.go [hunk 3]
```diff
@@ -40,6 +40,40 @@ func (e *IpIntel) Lookup(ctx context.Context, input *IpLookupRequest) (*pangea.P
...
+// Example:
+//
+//	 input := &ip_intel.IpGeolocateRequest{
+//	     Ip: "93.231.182.110",
+//	     Raw: true,
+//	     Verbose: true,
+//	     Provider: "crowdstrike",
+//	 }
+//
+//		checkOutput, _, err := ipintel.geolocate(ctx, input)
```
Comment (r3dcrosse) [inline pangea-sdk/service/ip_intel/api.go:56]: I think we can remove the extra tab here? Unless this was something `gofmt` did?
```suggestion
//	checkOutput, _, err := ipintel.geolocate(ctx, input)
```

**Comment by pangea-andrest** [comment]
Updated all @r3dcrosse comments. Thanks!

**Review by r3dcrosse — APPROVED** [review]

**Pull request merged**